experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1564,10 +1597,16 @@ def _validate_feature_configs(self, data):
                     continue
 
             elif NimbusExperiment.Application.is_mobile(application):
+                feature_config = feature_value_data["feature_config"]
+                blob = feature_value_data["value"]
+                schemas_in_range = feature_config.get_versioned_schema_range(
+                    min_version, max_version
+                )
```

Comment:
We're repeating this later on 1640. Lets instead pass the min version and max version into `_validate_with_fml` (like how `_validate_feature_against_schema` does) and call `get_versioned_schema_range` there.